test/main.tf

```diff
@@ -0,0 +1,67 @@
+# Basic terraform tests. If they break, you probably have introduced a breaking change.
+# These tests cannot catch errors, but can be used to verify that the more complex variables worked as expected.
+
+provider "aws" {
+  region  = var.region
+  profile = var.profile
+}
+
+# Defaults, except no multi-region cloudtrail to avoid hitting 5 trail limit
+module "test_module_1" {
```

Comment:
Give these meaningful names, e.g., `module.test` and `module.test_all`, assuming the goals are a minimal test and a maximal test.

Reply:
done